### DISCUSSION TITLE
Actividad en Canvas - Gestión de la configuración - Contabilidad del estado de la configuración e Identificación de la configuración

### DISCUSSION THREAD

**Discussion by fnavarroTEC** [Tec-de-Monterrey-Capstone-Project-2024/SpringBoot_Backend#62] [Announcements]
Estimado equipo buena tarde, les saludo por este medio.
Les adelanto la solicitud de generar entregable similar al de la sesión del pasado viernes 12 de abril, con evidencias de print screen dede Git Hub que me demuestre, junto con narrativa que describa cada evidencia dos tareas de la gestión de la configuración:
1- Contabilidad del estado de la configuración 
2- Identificación de la configuración 

Se genera en equipo y la suben individualmente.
Fecha de entrega Miercoles 17 de abril, fin del dia como limite.

Quedo a su señal, comparto instrucción tambien por canvas correo, 

FAvor de reponder de recepción por parte del lider o lideres de cada equipo, 

Saludos, Profesor Federico Navarro B.

**Comment by AntonioLaurance** [comment]
Buenas tardes profesor, confirmo recepción de instrucciones.

Saludos 
Ricardo Campos Luna
